server/repository/src/migrations/v2_04_00/remove_unique_description_on_tmp_breach.rs

```diff
@@ -0,0 +1,44 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "remove_unique_description_on_tmp_breach"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        if cfg!(feature = "postgres") {
+            sql!(
+                connection,
+                r#"
+                    ALTER TABLE temperature_breach_config DROP CONSTRAINT temperature_breach_config_description_key;
```

Comment:
I'm pretty sure this is ok but but I'm not 100% sure about an old version of posgres for example.
Officially you're supposed to look up the constraint name first then run a command line this.
Tempted to also modify the original table migration so any new installs won't ever get this broken version?

Reply:
Added a warn message just incase there's an issue hopefully with postgres people will be able to login and fix this if there was a problem...